app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyHomeDetailScreen.kt

```diff
@@ -0,0 +1,174 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
+
+import androidx.compose.animation.Crossfade
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.kks.nimblesurveyjetpackcompose.R
+import com.kks.nimblesurveyjetpackcompose.model.Survey
+import com.kks.nimblesurveyjetpackcompose.ui.theme.BlackRussian
+import com.kks.nimblesurveyjetpackcompose.ui.theme.NeuzeitFamily
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White70
+import com.kks.nimblesurveyjetpackcompose.util.TWEEN_ANIM_TIME
+import com.ramcosta.composedestinations.annotation.Destination
+
+@OptIn(ExperimentalPagerApi::class)
+@Destination
+@Composable
+fun SurveyHomeDetailScreen(survey: Survey) {
+    var currentPage by remember { mutableStateOf(0) }
+
+    Box(modifier = Modifier.fillMaxSize()) {
+        AsyncImage(
+            model = survey.coverImageUrl,
+            contentDescription = stringResource(id = R.string.survey_detail_background_image),
+            modifier = Modifier.fillMaxSize(),
+            placeholder = painterResource(id = R.drawable.ic_launcher_background),
+            contentScale = ContentScale.Crop
+        )
```

Comment:
What if we were to replace `model` with the full sized image (`url` + `l`) and replace `placeholder` with an `AsyncImagePainter` that loads the low res image? 
From the [Postman collection](https://documenter.getpostman.com/view/11835486/UVsPNPfU#ca6a12cc-f734-4d37-b28a-0655e00e868d):
> 💡 To get the high-resolution of an image just append “l” to the image URL obtained in the API response. 
```suggestion
        val painter = rememberAsyncImagePainter(model = survey.coverImagePlaceholderUrl)
        AsyncImage(
            model = survey.coverImageFullUrl,
            contentDescription = stringResource(id = R.string.survey_detail_background_image),
            modifier = Modifier.fillMaxSize(),
            placeholder = placeholderPainter,
            contentScale = ContentScale.Crop
        )
```
And in `Survey.kt`
```kotlin
data class Survey(
    val id: String,
    val coverImagePlaceholderUrl: String,
    val title: String,
    val description: String
) {
    val coverImageFullUrl: String
        get() = "${coverImagePlaceholderUrl}l"
}
```

Reply:
Tried adding `l` but it doesn't seem to fetch high-resolution one 🤔